config/routes.rb

```diff
@@ -65,6 +65,9 @@
   get "/t/:tag" => "home#tagged", :as => "tag"
   get "/t/:tag/page/:page" => "home#tagged"
 
+  get "/domain/:fqdn" => "home#for_domain", :as => "for_domain"
```

Comment:
Been thinking about it. Because we do normalization like strip off `www` and don't include the trailing `.` this isn't quite an FQDN. Rather than get technically nitpicky with PQDN, let's just call the field `domain`, similar to how `Tag` has `tag`.